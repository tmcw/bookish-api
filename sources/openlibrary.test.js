var OpenLibrary = require("./openlibrary");

test("OpenLibrary", async () => {
  const ol = new OpenLibrary();
  expect(ol).toBeTruthy();
  expect(await ol.ISBN("0812993543")).toMatchSnapshot();
});
