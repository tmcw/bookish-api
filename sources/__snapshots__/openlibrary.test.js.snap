// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpenLibrary 1`] = `
Object {
  "alibris_id": Array [
    "9780812993547",
  ],
  "amazon": Array [
    "0812993543",
  ],
  "goodreads": Array [
    "25489625",
  ],
  "isbn10": Array [
    "0812993543",
  ],
  "isbn13": Array [
    "9780812993547",
  ],
  "lccn": Array [
    "2015008120",
  ],
  "oclc": Array [
    "923007023",
  ],
  "openlibrary": Array [
    "OL25773328M",
  ],
}
`;
